src/languages/python.js

```diff
@@ -12,12 +12,11 @@ export default function(hljs) {
       'and as assert async await break class continue def del elif else except finally for ' +
       'from global if import in is lambda nonlocal|10 not or pass raise return try while with yield',
     built_in:
-      'abs delattr hash memoryview set all dict help min setattr any dir hex next slice ascii ' +
-      'divmod id object sorted bin enumerate input oct staticmethod bool eval int open str ' +
-      'breakpoint exec isinstance ord sum bytearray filter issubclass pow super bytes float ' +
-      'iter print tuple callable format len property type chr frozenset list range vars ' +
-      'classmethod getattr locals repr zip compile globals map reversed __import__ complex ' +
-      'hasattr max round',
+      '__import__ abs all any ascii bin bool breakpoint bytearray bytes callable chr classmethod ' +
+      'compile complex delattr dict dir divmod enumerate eval exec filter float format frozenset ' +
+      'getattr globals hasattr hash help hex id input int isinstance issubclass iter len list ' +
+      'locals map max memoryview min next object oct open ord pow print property range repr ' +
+      'reversed round set setattr slice sorted staticmethod str sum super tuple type vars zip',
```

Comment:
While we're here can we please just expand these to one-per-line (see) in an array (see `ecmascript.js` for an example) and then join them for keywords for future maintainability...

Reply:
How’s this?

Reply:
Awesome.  Just those two tiny changes I mentioned and then I think this is good to ship!